ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListConnectorsExecutor.java

```diff
@@ -118,20 +118,20 @@ private static SimpleConnectorInfo fromConnectorInfoResponse(
     return new SimpleConnectorInfo(
         name,
         info.type(),
-        info.config().get(ConnectorConfig.CONNECTOR_CLASS_CONFIG),
+        info.config().get(DescribeConnectorExecutor.CONNECTOR_CLASS_CONFIG),
```

Comment:
nit: can we centralize this constant somewhere within `ConfigInfos` (or wherever seems appropriate), instead of in the various places where it's being used?